docs/api/process.md

```diff
@@ -157,6 +158,21 @@ Returns `Object`:
 
 Returns an object with V8 heap statistics. Note that all statistics are reported in Kilobytes.
 
+### `process.getProcessMemoryInfo()`
+
+Returns `Object`:
+
+* `residentSet` Integer _Linux_ and _Windows_ - The amount of memory 
+currently pinned to actual physical RAM.
+* `private` Integer - The amount of memory not shared by other processes, such as
+  JS heap or HTML content.
+* `shared` Integer - The amount of memory shared between processes, typically
+  memory consumed by the Electron code itself.
```

Comment:
Would be good to add (redundant) information on each of these property descriptions saying they're in KB, the note below is easy to miss.